modules/ROOT/examples/live-demos/getusers-api-alternative/index.html

```diff
@@ -0,0 +1,16 @@
+<textarea id="getusers-api-alternative">
+<p>Existing at mention: <span class="mention" data-mce-mentions-id="user-3">@Alice Smith</span></p>
+<p>Add some more at mentions here to track them. Click the <strong>getUsers</strong> button in the toolbar to add a list of currently inserted users to the editor.</p>
+
+<br><br><br>
+
```

Comment:
```suggestion
<p>The deprecated <code>getUsers</code> API would return an array of currently mentioned names which were mentioned since loading the editor.</p>
<p>Add some mentions to the content by typing "@a...", then click the <strong>getUsers</strong> button in the toolbar to insert a list of names which have been mentioned.</p>

<br><br><br>

```
Only mentions added by the user will be returned.